libsolidity/ast/Types.h

```diff
@@ -673,6 +673,7 @@ class ContractType: public Type
 	}
 
 	bool isSuper() const { return m_super; }
+	bool isPayable() const;
```

Comment:
Please add a comment about this returning true if and only if there is a payable fallback function.